lib/models/addon.js

```diff
@@ -637,11 +637,11 @@ Addon.prototype.jshintAddonTree = function() {
   }
 
   var addonJs = this.addonJsFiles(addonPath);
-  var jshintedAddon = this.app.addonLintTree('addon', addonJs);
+  var lintTrees = this.eachAddonInvoke('lintTree', addonJs);
 
-  return new Funnel(jshintedAddon, {
```

Comment:
Do we need this Funnel still?  Unless `lintTree` returns the files in `<current-addon-name>/tests` it seems that we do need it...